utils/ort/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -209,8 +209,8 @@ fun OkHttpClient.download(url: String, acceptEncoding: String? = null): Result<P
     }.mapCatching { response ->
         val body = response.body
 
-        if (!response.isSuccessful || body == null) {
-            body?.close()
+        if (!response.isSuccessful) {
+            body.close()
```

Comment:
Why this change? Does AWS allow null bodies on success? If so, this should be extracted to a preceding preparing refactoring commit, that thoroughly explains this in the commit message.

Reply:
Ah, wait, this is due to the OkHttp alpha upgrade, and thus should be dropped together with the upgrade.

Reply:
Fixed with refactor.

Reply:
Only in the first commit (won't compile otherwise), gone in the HEAD.